tests/test_decodepsbtqr.py

```diff
@@ -283,8 +283,6 @@ def test_short_4_letter_mnemonic_qr():
 
 def test_bitcoin_address():    
     bad1 = "loremipsum"
-    bad2 = "0xde0b295669a9fd93d5f28d9ec85e40f4cb697bae"
-    bad3 = "121802020768124106400009195602431595117715840445"
```

Comment:
Rather than removing these, should they instead be actually used as negative tests? Not sure when this test was introduced but the omission looks like an oversight.

Reply:
if those are good examples to be used in negative tests, the lines can also be just commented out for later usage... would prefer not to add any newly python code (tests) to the PR as there is lots of stuff to review already.

Reply:
For now I'd go that route (keep but comment out) and maybe drop a `# TODO:` comment in there so we don't forget to do something with them eventually.